src/i965_device_info.c

```diff
@@ -440,6 +441,7 @@ static struct hw_codec_info kbl_hw_codec_info = {
     .render_init = gen9_render_init,
     .post_processing_context_init = gen9_post_processing_context_init,
     .max_resolution = gen9_max_resolution,
+    .preinit_hw_codec = gen9_hw_codec_preinit,
```

Comment:
Actually there is a hidden bug when creating a VA config with VAConfigAttribRateControl. The current driver can do CBR and VBR even if the corresponding attribute is not returned to user via  VAConfigAttribRateControl.